src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -4,48 +4,61 @@
 <%@ page import="java.util.List" %>
 <%@ page import="codeu.model.data.Profile" %>
 
-<%
-Profile profile = (Profile) request.getAttribute("about");
-%>
-
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
-<title>UserProfile</title>
+<title><%= request.getSession().getAttribute("user") %></title>
 </head>
 <body>
 
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/profile">Profile</a>
-    <% if(request.getSession().getAttribute("user") != null){ %>
+      <% if(request.getSession().getAttribute("user") != null){ %>
       <a></a>
-    <% } else{ %>
-      <a href="/login">Login</a>
-    <% } %>
+      <% } else{ %>
+        <a href="/login">Login</a>
+      <% } %>
     <a href="/about.jsp">About</a>
   </nav>
   
-  <h1>This is Exuser's Page</h1>
- <div id="container">
-
+  <div id="container">
     <% String username = (String) request.getAttribute("username");
-    	if(request.getSession().getAttribute("user") != null && username.equals("exuser")){ %>
-      <h1>About Me!</h1>
-            
-      <form>
-      	Edit Text:<br>
-          <input type = "text" about = "about">
-          <br>        
-        <button type="submit">Update</button>
-      </form>
-
+ 	if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hi my name is <%= request.getSession().getAttribute("user") %>!</a>
+      <p>About me:</p>
       <hr/>
-    <% } %>
-
-  </div>
+    <%} %>
+    
+  <% List<Profile> profiles = (List<Profile>) request.getAttribute("profiles");
+    if(profiles == null){
+  %>
+    <p>Create a profile to get started.</p>
+  <% } else{
+  %>
+    <%
+      for(Profile profile : profiles){
+      String user = (String) request.getSession().getAttribute("user");
+    %>
+  
+      <% if(profile.getProfile().equals(user)){
+      %>
+        <a><%= profile.getAbout() %></a>
```

Comment:
I think you should use `profileStore.get{rofile(username)` here, since that handles the logic for you.
It might be better if the logic is handled in the servlet instead of in the jsp,

Reply:
Noted!